PINCE.sh

```diff
@@ -16,12 +16,18 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 '
 
+if [ ! -d ".venv/PINCE" ]; then
+	echo "Please run \"sh install_pince.sh\" first!"
```

Comment:
People won't be able to directly run PINCE.sh after fresh clone anymore with this change, which would happen sometimes.